azkaban-web-server/src/main/java/azkaban/flowtrigger/plugin/FlowTriggerDependencyPluginManager.java

```diff
@@ -229,4 +253,79 @@ public void shutdown() {
       }
     }
   }
+
+  /**
+   * A parent-last classloader that will try the child classloader first and then the parent.
+   */
+  private static class ParentLastURLClassLoader extends ClassLoader {
```

Comment:
Do you think this should be pulled out into its own file? I'm not sure - what's your intuition?

Reply:
yeah, i think this class is too big to treated as inner class, move it out. Thanks